### ISSUE TITLE
Many contributor photos no longer appear

### ISSUE THREAD

**Issue by PaulHeggarty** [lingdb/Sound-Comparisons#402]
At 
http://www.soundcomparisons.com/#/Contributors
many images that used to work no longer do, and just show the dummy placeholder...

**Comment by AvivaShimelman** [comment]
Let me know if there's anything I should do.

A.

On 9/15/16, Paul Heggarty <notifications@github.com> wrote:
> At
> http://www.soundcomparisons.com/#/Contributors
> many images that used to work no longer do, and just show the dummy
> placeholder...
>
>
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/lingdb/Sound-Comparisons/issues/402


-- 
Aviva Shimelman, PhD

**Comment by runjak** [comment]
Hm - I may be unable to fix this:
I don't have pictures for the other contributors either on my machine or on the server.
It also appears that these have never been added properly to the GitHub [repository](https://github.com/lingdb/Sound-Comparisons/tree/master/img/contributors) like the other images. If these images could be made available by upload into the repository or to the server it would be a good step towards solving this problem.

**Comment by AvivaShimelman** [comment]
Happy to do whatever falls to me on this, but I'm not sure what that
is. Do you need me to re-upload? Re-name? ??

A.

On 9/15/16, Jakob Runge <notifications@github.com> wrote:
> Hm - I may be unable to fix this:
> I don't have pictures for the other contributors either on my machine or on
> the server.
> It also appears that these have never been added properly to the GitHub
> [repository](https://github.com/lingdb/Sound-Comparisons/tree/master/img/contributors)
> like the other images. If these images could be made available by upload
> into the repository or to the server it would be a good step towards solving
> this problem.
>
> --
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/lingdb/Sound-Comparisons/issues/402#issuecomment-247248707


-- 
Aviva Shimelman, PhD

**Comment by runjak** [comment]
@AvivaShimelman if you have the contributor photos that are named after the initials (like `PAH.jpg` or `AM.jpg`) it would be perfect if you could upload these again. The list of currently working contributor images is found here: […/img/contributors](https://github.com/lingdb/Sound-Comparisons/tree/master/img/contributors).
I'll then make sure to save them to GitHub as well.

**Comment by PaulHeggarty** [comment]
correction to the last comment.  The problem is not primarily with speaker photos, but other contributors.  This is for me to fix.

**Comment by AvivaShimelman** [comment]
Internet, apparently, too slow right now to connect to github. It
sometimes kicks in a bit after 11. If I still can't get to it after
that, I'll ask you to just send the list.
Not sure what you mean by the initials, but I'm assuming it will
become clear when I see the list.

I'll upload them, if I can, to a folder that's pretty clearly labeled
on oc "Malakula"

A.

On 9/15/16, Jakob Runge <notifications@github.com> wrote:
> @AvivaShimelman if you have the contributor photos that are named after the
> initials (like `PAH.jpg` or `AM.jpg`) it would be perfect if you could
> upload these again. The list of currently working contributor images is
> found here:
> […/img/contributors](https://github.com/lingdb/Sound-Comparisons/tree/master/img/contributors).
> I'll then make sure to save them to GitHub as well.
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/lingdb/Sound-Comparisons/issues/402#issuecomment-247266543


-- 
Aviva Shimelman, PhD

**Comment by AvivaShimelman** [comment]
OK. I just did get into github. But now I'm thinking that it's moot.
The image files with initials in the file names seem to be mug shots
of SC-ers. Are these place-holders? A joke?

A.

On 9/15/16, Paul Heggarty <notifications@github.com> wrote:
> correction to the last comment.  The problem is not primarily with speaker
> photos, but other contributors.  This is for me to fix.
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/lingdb/Sound-Comparisons/issues/402#issuecomment-247269196


-- 
Aviva Shimelman, PhD

**Comment by runjak** [comment]
Yes, this is precisely about the mug shots of SC-ers - not about the speaker images.
These images are to be displayed on the soundcomparisons contributors page: http://soundcomparisons.com/#/Contributors

**Comment by LauraWae** [comment]
Hi!
We cannot see AAviva's photo on the contributors' page, but besides her name, the photo of Anton Stepikhov appears. I would be very happy to see the person there with whom I have been working now for more than a year, and I suppose, she would so, too.
So, thanks!

**Comment by AvivaShimelman** [comment]
Will find.

A.

On 3/10/17, LauraWae <notifications@github.com> wrote:
> Hi!
> We cannot see AAviva's photo on the contributors' page, but besides her
> name, the photo of Anton Stepikhov appears. I would be very happy to see the
> person there with whom I have been working now for more than a year, and I
> suppose, she would so, too.
> So, thanks!
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/lingdb/Sound-Comparisons/issues/402#issuecomment-285650088


-- 
Aviva Shimelman, PhD

**Comment by PaulHeggarty** [comment]
Some of these, including the three candidates for AS initials, have now been fixed.

**Comment by AvivaShimelman** [comment]
"AS" initials?
In any case, I'll upload all the photos I have left today.

A.

On 3/20/17, Paul Heggarty <notifications@github.com> wrote:
> Some of these, including the three candidates for AS initials, have now been
> fixed.
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/lingdb/Sound-Comparisons/issues/402#issuecomment-287851931


-- 
Aviva Shimelman, PhD
